src/config/constants.js

```diff
@@ -58,8 +57,22 @@ export const DEFAULT_MEASURING_ARROW_WIDTH = 95;
 export const MIN_MEASURING_ARROW_WIDTH = 47.5;
 export const MAX_MEASURING_ARROW_WIDTH = 475;
 export const MEASURING_ARROW_STEP = 47.5;
+export const MEASURING_ARROW_TEXT_FONT_SIZE = 16;
+// CharCode 8982 is the crosshair indicating draggability
+export const MEASURING_ARROW_SYMBOL_CHAR_CODE = 8982;
 // every 47.5 pixels === 100nm
 export const MEASURING_ARROW_UNITS_TO_NANOMETER_CONVERSION_FACTOR = 100 / 47.5;
+// used to initially center measuring arrow labels
+export const APPROXIMATE_MEASURING_ARROW_TEXT_WIDTH = 49;
+export const APPROXIMATE_MEASURING_ARROW_DRAG_ICON_WIDTH = 17.5;
+
+// constants used in utils/physics, to calculate wavelength given frequency
+export const SPEED_OF_LIGHT_IN_METERS_PER_SECOND = 3 * 10 ** 8;
```

Comment:
If you just want to use scientific notation, you can always do `3e8` or `1e-9`.

Reply:
oh nice did not know JS supported this